inclusive-design/wecount.inclusivedesign.ca
# Increase size of funder logos (resolves #342)

## Description
This PR increases the size of funder logos in the website footer and removes the link in the Hewlett funder statement.


## Steps to test

1. Go to deploy preview
2. Scroll down to footer
3. See funder section of footer

**Expected behavior:** 
No links in funder statements and the width of the funder logos should span the width of the funder statement.

## Additional information
N/A


## Related issues
resolves #342
